x-pack/plugins/security_solution/public/management/cypress/e2e/serverless/feature_access/complete.cy.ts

```diff
@@ -22,7 +22,7 @@ describe(
         // how to enable experimental features in the Cypress tests.
         // kbnServerArgs: [
         //   `--xpack.securitySolution.enableExperimental=${JSON.stringify([
-        //     'featureFlagName',
+        //     'crowdstrikeRunScriptEnabled',
```

Comment:
Do you need this change?

Reply:
Good catch. This can be reverted for the purposes of this PR, will be needed for another one.